### PR TITLE
docs: update chinese translations

### PR DESCRIPTION
**Pull request by lodisy**
Closes the original PR
https://github.com/t3-oss/create-t3-app/pull/965

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Comment by Fatimah1986** [comment]
Yes

**Comment by juliusmarminge** [comment]
Do you know anyone for review?

**Comment by lodisy** [comment]
> Do you know anyone for review?

@fernandoxu

File: www/src/pages/zh-hans/why.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+我们创建 create-t3-app 的初衷，是因为 [Theo](https://twitter.com/t3dotgg) 拒绝将他最喜欢的几种技术做成模板。受到 create-next-app、[Astro 的 CLI 命令行工具](https://astro.build) 和对类型安全普遍热爱的启发，create-t3-app 团队努力地为创建新的 T3 Stack 项目建立了最好的起点。
+
+如果你对以类型安全的方式使用 Next.js 感兴趣，这就是开始的地方。如果你对我们的具体技术选择感到好奇，请继续阅读 :)
+
+## 为什么使用 TypeScript？
+
+JavaScript 已经很难了。为什么还要增加更多规则？
+
+我们坚信 TypeScript 提供的体验会帮助你成为更好的开发者。在你写代码时，它通过预设的数据类型来提供实时反馈；而如果你在代码中尝试访问不存在的属性或尝试传递错误类型的值时，它要么会在编辑器中提供有用的自动补全功能，要么会用红色的波浪线来提醒你错误代码，让你不得不进行进一步调试。无论你是 web 开发的新手还是老手，TypeScript 的“严格性”可以让它比原生 JavaScript 为你带来更少的挫败感，却带来更一致的开发体验。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/why.md:16]: 体验->经验 ？

File: www/src/pages/zh-hans/why.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+类型安全可以让你更快地写代码。如果你不承认这一点，你可能 [用错了 TypeScript。。。](https://www.youtube.com/watch?v=RmGHnYUqQ4k)
+
+## 为什么使用 Next.js？
+
+我们热爱 React。它使 UI 开发以我们从未想象过的方式进行。但它也会将开发者引向一些艰难的道路。
+
+Next.js 为使用 React 创建应用程序提供了一种轻 opinionated、高度优化的方法。从路由到 API 定义，再到图片渲染，我们都相信 Next.js 能引导开发者做出正确的决定。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/why.md:24]: 轻opinionated -> 轻度/略微武断？

File: www/src/pages/zh-hans/why.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Prisma 之于 SQL 就像 TypeScript 之于 JS。它创造了前所未有的开发者体验。通过从与 [几种数据库](https://www.prisma.io/docs/concepts/database-connectors) 兼容的自定义 schema 中生成类型，Prisma 保证了从数据库到应用程序的端到端类型安全。
+
+Prisma 提供了 [一整套工具](https://www.prisma.io/docs/concepts/overview/should-you-use-prisma#-you-want-a-tool-that-holistically-covers-your-database-workflows)，使与数据库的日常交互更加容易。值得注意的是，用 Prisma 客户端来负责数据库查询，使得 SQL 变得十分简单，以至于你几乎不会注意到你在使用它，而 Prisma Studio 是一个方便的数据库 GUI，可以让你快速读取和操作数据，而无需编写代码。
+
+### Tailwind CSS
+
+Tailwind 给人的感觉就像“禅式的 CSS”。
+
+Tailwind 通过良好的默认颜色、间距和其他 primitive 的形式为我们提供了构建块，使得创建漂亮美观的应用程序变得十分容易。而当你想将应用程序提升到一个新的水平并创造美丽而独特的 UI 时，不同于组件库，它并不会在这方面阻碍你。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/why.md:44]: primitive -> 原语？常见的计算机词汇

File: www/src/pages/zh-hans/folder-structure.mdx [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+<Form />
+
+<Diagram />
+
+<div data-components="prisma">
+
+### `prisma`
+
+文件夹 `prisma` 包含了 `schema.prisma` 文件，它被用于配置数据库连接和定义数据结构。数据库的迁移记录文件以及初始化脚本（如果你需要的话）也会被放在这个文件夹里。 参看 [Prisma 的用法](/zh-hans/usage/prisma) 来了解更多。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/folder-structure.mdx:21]: 初始化脚本 -> 种子脚本？种子也是后端普遍的叫法吧，可以保留

File: www/src/pages/zh-hans/folder-structure.mdx [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+### `src/server`
+
+文件夹 `server` 被用于清晰划分服务端和客户端代码。
+
+</div>
+<div data-components="nextauth+trpc">
+
+### `src/server/common`
+
+文件夹 `common` 包含了多为可复用的服务端代码。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/folder-structure.mdx:77]: 多为好像不太合适，通用的/常用的吧。一般也不放别的代码

File: www/src/pages/zh-hans/faq.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+---
+title: 常见疑问
+description: 关于 Create T3 App 的常见疑问
+layout: ../../layouts/docs.astro
+lang: zh-hans
+---
+
+这里罗列了一些关于 `create-t3-app` 的常见疑问。
+
+## 下一步呢？我怎么通过 `create-t3-app` 创建应用？
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/faq.md:10]: 制作吧，到这应该不是创建了

File: www/src/pages/zh-hans/faq.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+## 项目中为什么会有 `.js` 格式的文件？
+
+正如在 [T3 - 原则第三条](/zh-hans/introduction#类型安全不是可选的) 里所讲到的，我们将类型安全放在首位。不幸的是，并非所有的框架和插件都支持 TypeScript，这也就意味着有一些配置文件必须是 `.js` 格式。
+
+我们想要强调的是，这些文件是 JavaScript 是有原因的，我们通过根据它们所使用的库的支持程度来显式声明每个文件的类型（`cjs` 或 `mjs`）。而且，项目中所有的 `js` 文件也会通过使用 `@ts-check` 注解来进行类型检查。
+
+## 我正努力为我的应用添加多语言功能。有我能够借鉴的吗？
+
+我们已经决定 `create-t3-app` 默认不支持多语言功能，因为这是一个非常 opinionated 的话题，有很多不同的方法来实现它。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/faq.md:53]: 又一个opinionated😂，这个词还是翻了吧

File: www/src/pages/zh-hans/introduction.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+算是吧？`create-t3-app` 是由经验丰富的 T3 Stack 开发者构建的 CLI 命令行工具，用于简化一个模块化的 T3 Stack 应用程序的配置过程。这意味着每个部分都是可选的，而“模板”则是根据你的具体需求生成的。
+
+经过无数项目和多年的技术积累，我们沉淀了很多的观点和见解。我们尽力将它们融入到了这个 CLI 中。
+
+这**不是**一个包罗万象的模板。我们**期望**你能带来自己的库，以解决**你自己的**应用程序需求。虽然我们不想为更具体的问题（如状态管理和项目部署）指定解决方案，但我们 [在这里列出了一些推荐选项](/zh-hans/other-recs)。
+
+## T3 原则
+
+坦白地说——这是一个非常 _opinionated 的项目_。我们在构建方面有不少共同的核心信念，并把它们作为我们做出决策的基础。
```
[the remark this answers is not in the document]
Comment (lodisy) [reply www/src/pages/zh-hans/introduction.md:30]: 应该是非常主观 个人特色的 融入个人想法的

单单固执己见这一个词感觉偏离了意思 就不如不翻译

File: www/src/pages/zh-hans/why.md (hunk 2 above)
Comment (lodisy) [reply www/src/pages/zh-hans/why.md:24]: 同第一点

File: www/src/pages/zh-hans/usage/next-js.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/W4UhNo3HAMw" title="Next.js is a backend framework" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+不妨观看 [Theo 在 Next.js Conf 上的演讲](https://www.youtube.com/watch?v=W4UhNo3HAMw) 来更好地理解 Next.js 究竟是什么以及它是如何运作的。</p>
+
+## 为什么我应该使用它？
+
+我们热爱 React。它以一种我们从未想过的方式将 UI 开发变得如此简单。但是它同样也给开发者们带来了一些困难。而 Next.js 就为我们提供了一种轻量 opinionated，却高度优化的方式来使用 React 创建应用。从路由到 API 定义，再到图片渲染，我们相信 Next.js 会帮助开发者做出正确的决定。
+
+而将 Next.js 搭配 [Vercel](https://vercel.com/) 一同食用使得 web 应用的开发体验前所未有的棒。他们慷慨的免费套餐以及非常直观的用户界面让我们可以通过点击几下的方式就可以部署网站（我们 ❤️ Vercel）。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/usage/next-js.md:20]: 食用是typo还是故意这么翻译呢

Comment (lodisy) [reply www/src/pages/zh-hans/usage/next-js.md:20]: 😂 放飞自我了

File: www/src/pages/zh-hans/usage/trpc.md [hunk 10]
```diff
@@ -0,0 +1,324 @@
...
+  return createNextApiHandler({
+    router: appRouter,
+    createContext,
+  })(req, res);
+};
+
+export default handler;
+```
+
+### 积极的界面更新
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/usage/trpc.md:258]: **乐观更新** 应该是更普遍的叫法

File: www/src/pages/zh-hans/usage/prisma.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+当你选择 NextAuth.js 搭配 Prisma 使用时，根据 [NextAuth.js 文档](https://next-auth.js.org/adapters/prisma)，schema 文件会被自动生成，其中默认包含了 `User`、`Sessiong`、`Acount` 和 `VerificationToken` 模型，并已经自动添加了推荐的字段。
+
+## 默认数据库
+
+默认数据库是一个 SQLite 数据库，这对开发阶段来说很棒，因为它能够快速启动概念验证，但不推荐用于生产环境。你可以将 `datasource` 中的 `provider` 修改为 `postgresql` 或 `mysql`，然后在环境变量里修改一下对应数据库的连接地址，来更换数据库。
+
+## 给数据库填充初始数据
+
+[给数据库填充初始数据](https://www.prisma.io/docs/guides/database/seed-database) 是一种。。来快速给你的数据库填充测试数据来帮助你开始。为了设置填充，你将需要在目录 `/prisma` 下创建一个 `seed.ts` 脚本文件，然后在 `package.json` 文件中添加 `seed` 命令。你也会需要一个 TypeScript 执行环境来运行该数据填充脚本。我们推荐 [tsx](https://github.com/esbuild-kit/tsx)，它采用 esbuild，是一个性能很好的 TypeScript 执行环境，无需额外的 ESM 配置，不过也可以使用 `ts-node` 或其他执行环境。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/usage/prisma.md:28]: 是一种很好的方法，

File: www/src/pages/zh-hans/usage/next-auth.md [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+    } & DefaultSession["user"];
+  }
+}
+```
+
+你也可以用同样的方法将其他数据，例如 `role` 字段加入到 `session` 对象里，**但不应该被滥用于在客户端存储敏感数据**。
+
+## 搭配 tRPC 的用法
+
+当你搭配 NextAuth.js 和 tRPC 一同使用时，你可以通过 [中间件](https://trpc.io/docs/v10/middlewares) 来创建复用的、受保护的 procedure 路由函数。只有登入后的用户才能访问这些受保护的路由。`create-t3-app` 已经为你铺好了路，使你能够在认证路由里轻松访问到 session 里的数据。
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/usage/next-auth.md:70]: 可复用的

File: www/src/pages/zh-hans/usage/tailwind.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+  background-color: #fff;
+  border: 1px solid #e2e8f0;
+  border-radius: 0.25rem;
+  padding: 1rem;
+}
+```
+
+2. 在组件里导入 CSS 文件
+
+````jsx
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/usage/tailwind.md:33]: 多了一行```jsx，且多了一个`

File: www/src/pages/zh-hans/deployment/docker.md [hunk 14]
```diff
@@ -0,0 +1,213 @@
...
+## 有用的资源
+
+| 资源                                  | 链接                                                                 |
+| ------------------------------------- | -------------------------------------------------------------------- |
+| Dockerfile 参考手册                   | https://docs.docker.com/engine/reference/builder/                    |
+| Compose file 第三版参考手册           | https://docs.docker.com/compose/compose-file/compose-file-v3/        |
+| Docker CLI 命令行工具参考手册         | https://docs.docker.com/engine/reference/commandline/docker/         |
+| Docker Compose CLI 命令行工具参考手册 | https://docs.docker.com/compose/reference/                           |
+| 使用 Docker 镜像 部署 Next.js         | https://nextjs.org/docs/deployment#docker-image                      |
+| Next.js 搭配 Docker                   | https://benmarte.com/blog/nextjs-in-docker/                          |
```
Comment (fernandoxu) [inline www/src/pages/zh-hans/deployment/docker.md:211]: Docker 中的 Next.js

**Review by fernandoxu — COMMENTED** [review]
还少一篇[netlify](https://create.t3.gg/en/deployment/netlify)的翻译

**Comment by lodisy** [comment]
> 还少一篇[netlify](https://create.t3.gg/en/deployment/netlify)的翻译

这是最新的，已添加

**Review by fernandoxu — COMMENTED** [review]
💯

**Comment by c-ehrlich** [comment]
Ready to merge?

**Comment by lodisy** [comment]
> Ready to merge?

I guess so, we can still do some polish if needed later.

**Review by c-ehrlich — APPROVED** [review]
Thanks for all the work!! Lets get this merged :)

**Pull request merged**